packages/devtools_app/lib/src/code_size/file_import_container.dart

```diff
@@ -49,21 +50,20 @@ class _FileImportContainerState extends State<FileImportContainer> {
         Expanded(
           child: Container(
```

Comment:
let's use the `RoundedOutlinedBorder` widget from common_widgets.dart here

Reply:
Done